### PR TITLE
Full Standardization

### PR DESCRIPTION
**Pull request by Lord-Valen**
See #32, #31.

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -9,4 +9,4 @@
         .#__std.direnv_lib 2>/dev/null
 )"
 
-use std nix //nixt/devshells:default
+use std nix //app/devshells:default
```
Comment (blaggacao) [inline .envrc:12]:
```suggestion
use std cells //app/devshells:default
```

File: flake.nix [hunk 2]
```diff
@@ -11,29 +11,39 @@
 
     nixpkgs.url = "nixpkgs";
 
+    yants.url = "github:divnix/yants";
```
Comment (blaggacao) [inline flake.nix:14]: also available under `std.yants` (convenience)

File: flake.nix [hunk 3]
```diff
@@ -11,29 +11,39 @@
...
       ];
     }
-      {
-        packages = std.harvest self ["nixt" "packages"];
-        devShells = std.harvest self ["nixt" "devshells"];
-      };
+    {
+      packages = std.harvest self ["app" "packages"];
+      devShells = std.harvest self ["app" "devshells"];
+      lib = std.harvest self ["lib" "nixt"];
```
Comment (blaggacao) [inline flake.nix:47]: If you also want to expose `lib.nixt.inject` in addition to `lib.inject` (for whatever reason; maybe you don't).

Then, there's a trick to do that which goes like this:
`lib = std.harvest self [ ["lib"] ["lib" "nixt"]]`

In cases where the lib is system agnostic, which it looks like it _is_, you can use `std.pick`, instead.

**Review by blaggacao — COMMENTED** [review]
I'll do a sprint on the dream2nix quality-of-life integration, shortly

**Comment by Lord-Valen** [comment]
Best not. Yants will complain (I did not use it correctly). Beside that I've been tinkering with the API, and the CLI doesn't support this yet either.

File: .envrc (hunk 1 above)
Comment (Lord-Valen) [reply .envrc]: Fixed

File: flake.nix (hunk 3 above)
Comment (Lord-Valen) [reply flake.nix]: `std.pick` is now used. Is there any value in exposing lib.nix.inject? It's mainly intended for CLI use.

Comment (blaggacao) [reply flake.nix]: > value in exposing lib.nix.inject?

I don't think so. But I took the opportunity to spread the word about that little trick. :smile:

**Pull request merged**